.travis.yml

```diff
@@ -1,15 +1,21 @@
+dist: xenial
 language: python
-python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "3.6"
-  - "pypy"
+matrix:
+  fast_finish: true
+jobs:
+  include:
+    - python: '2.7'
+    - python: "2.7"
```

Comment:
@glow-mdsol `2.7` is duplicated...

Reply:
Addressed in https://github.com/mdsol/requests-mauth/pull/17 - Thanks @glow-mdsol!